tests/unit/amo/components/TestCategoryHead.js

```diff
@@ -65,4 +65,30 @@ describe(__filename, () => {
 
     expect(root.find('title')).toHaveText(category.name);
   });
+
+  it('renders a "description" meta tag when category description is available', () => {
+    const description = 'some description for a category';
+    const category = { ...fakeCategory, description };
+    const root = render({ category });
+
+    expect(root.find('meta[name="description"]')).toHaveLength(1);
+    expect(root.find('meta[name="description"]')).toHaveProp(
+      'content',
+      description,
+    );
+  });
+
+  it('does not render a "description" meta tag when category description is null', () => {
```

Comment:
I'm looking at api docs and it doesn't say this could be null. Usually I've seen string|null so just wanted to double check this :)

Reply:
it should say that given that before my patch on the server, all descriptions were null.. I'll update the API doc ☺️

Reply:
I filed: https://github.com/mozilla/addons-server/issues/9818